src/lib/tokenizer.js

```diff
@@ -0,0 +1,88 @@
+class Tokenizer {
+  static theTokenizer;
+
+  constructor(content, literalsList) {
+    this.program = content;
+    this.literals = literalsList;
+    this.tokens = [];
+    this.currentToken = 0;
+
+    this.tokenize();
+  }
+
+  tokenize() {
+    let tokenizedProgram = this.program;
+    tokenizedProgram = tokenizedProgram.replace(/\n/g, '');
+    console.log(this.program);
+
+    this.literals.forEach(
+        s => {
+          tokenizedProgram = tokenizedProgram.replace(s, `_${s}_`);
+          console.log(tokenizedProgram);
+        });
+    tokenizedProgram = tokenizedProgram.replace(/__/g, '_');
+    console.log(tokenizedProgram);
+    const temparray = tokenizedProgram.split('_');
+    this.tokens = temparray.slice(1);
```

Comment:
I forgot the fact that the second parameter of `slice()` method is an exclusive index, i.e. the last element won't be in the slice. I've fixed it now.